Commands.adoc

```diff
@@ -0,0 +1,226 @@
+= Available Commands and Examples
+:toc:
+:toc-placement!:
+
+Once you do `vagrant up` and run an ADB box, as a developer you may want to
```

Comment:
I'd drop "as a developer". Just you may want to or even better "you can". The latter is more positive and decisive.